.vscode/extensions.json

```
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "golang.go"
+    ]
+}
```

```suggestion
}

```

モダンな処理系では関係ないと思うけれど...．